openassessment/templates/openassessmentblock/grade/oa_grade_complete.html

```diff
@@ -135,7 +134,7 @@ <h3 class="message__title">{% trans "Your Feedback Has Been Submitted" %}</h3>
                                     <ol class="list list--fields submission__feedback__fields {{ has_submitted_feedback|yesno:"is--hidden," }}"
                                             {{ has_submitted_feedback|yesno:'aria-hidden=true,aria-hidden=false' }}>
                                         <li class="field field-group field--radio feedback__overall">
-                                            <h4 id="feedback__overall__prompt__{{ xblock_id }}" class="field-group__label">{% trans "Select the statements below that best reflect your experience with peer assessments" %}:</h4>
+                                            <h5 id="feedback__overall__prompt__{{ xblock_id }}" class="field-group__label">{% trans "Select the statements below that best reflect your experience with peer assessments" %}</h5>
```

Comment:
@staubina I think this was the place where we need a period (in place of the deleted colon?)